scripts/datacontainer/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM mcr.microsoft.com/oss/mirror/docker.io/library/ubuntu:20.04
```

Comment:
please update this to 22.04
This is updated in the pipeline file but missed here.
https://github.com/Azure/azure-rest-api-specs/blob/main/.azure-pipelines/azure-pipelines-data-container.yml

Reply:
Good call, thanks. I've made the change.